pom.xml

```diff
@@ -406,6 +407,11 @@
                 <artifactId>slf4j-api</artifactId>
                 <version>1.7.24</version>
             </dependency>
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-simple</artifactId>
+                <version>1.7.24</version>
```

Comment:
this is the same version as `slf4j-api` on line 408, isn'it? if so, such version number should be declared as a variable under `<properties>`